### ISSUE TITLE
doc: Scale RR diagrams to page column (enlarge on click)

### ISSUE THREAD

**Issue by msudberg** [MaterializeInc/materialize#8430]
Some of the longer RR diagrams don't fit well within our page layout's central column, either breaking into the right navigation or being broken up into multiple horizontal diagrams that can be hard to parse.  (See [`SELECT`](https://materialize.com/docs/sql/select/) for a particularly egregious example.)

To fix this, I would like to:

* Keep diagrams narrower than the central column width the same size
* Automatically scale down to column width any diagrams wider than the central column
* Pop up a full-size image (i.e., bigger than the central column) when you click or double-click on the scaled-down image

This won't make the longer diagrams completely vertical, but it should help.  They can at least have the entire page width.

**Comment by bobbyiliev** [comment]
Closing this issue as it was fixed by @joacoc in https://github.com/MaterializeInc/materialize/pull/15485 🎉
